lib/tasks/migrate/20230331222716_create_tipline_newsletters.rake

```diff
@@ -0,0 +1,30 @@
+namespace :check do
+  namespace :migrate do
+    task create_tipline_newsletters: :environment do
+      TeamBotInstallation.where(user: BotUser.smooch_user).find_each do |tbi|
+        team = tbi.team
+        tbi.get_smooch_workflows.to_a.each do |workflow|
+          language = workflow['smooch_workflow_language']
+          unless workflow['smooch_newsletter'].blank?
+            newsletter = TiplineNewsletter.new
+            newsletter.introduction = workflow.dig('smooch_newsletter', 'smooch_newsletter_introduction')
+            newsletter.rss_feed_url = workflow.dig('smooch_newsletter', 'smooch_newsletter_feed_url')
+            articles = workflow.dig('smooch_newsletter', 'smooch_newsletter_body').to_s.split("\n\n")
+            newsletter.first_article = articles[0]
+            newsletter.second_article = articles[1]
+            newsletter.third_article = articles[2]
+            newsletter.number_of_articles = [[workflow.dig('smooch_newsletter', 'smooch_newsletter_number_of_articles').to_i, articles.size].max, 3].min
```

Comment:
a little confused by this line - if I were to state it another way, would it be:
```
workflow.dig('smooch_newsletter', 'smooch_newsletter_number_of_articles').to_i ||
   articles.size ||
   3
```
?

or is there a case where we would expect something to return greater than 3 and want to cap it?

Reply:
The latter. We currently may have in the database newsletters that have more than 3 articles, but now we are going to support at maximum 3. So when we migrate, newsletters that have more than 3 articles will be set to 3.